src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,300 @@
+    }
+
+    /**
+     * Prints out a message when all the task is unable to be exported and saved.
+     *
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message exportTaskErrorMessage() {
+        return new Message("Unable to save task.");
+    }
+
+    /**
+     * Prints out a message if task from previous interactions is loaded.
+     *
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message loadTaskSuccessMessage() {
+        return new Message("I have loaded in task(s) saved from our previous interactions.");
+    }
+
+    /**
+     * Prints out a message when the task cannot be imported.
+     *
+     * @param description Description of task to be imported.
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message importTaskErrorMessage(String description) {
+        return new Message(String.format("Cant import %s", description));
+    }
+
+    /**
+     * Prints out a message if this is the first time the user is using duke.Duke.
+     *
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message importTaskErrorMessage() {
+        return new Message("This are the commands that I recognised:",
+                "bye - Ends the chat session.",
+                "todo <description> - Adds a new todo to the task list.",
+                "deadline <description> /by <date/time> - Adds a new deadline to the task list",
+                "event <description> /at <date/time> - Adds a new event to the task list",
+                "list - return a list of all the task",
+                "done <number> - Sets the task to be done",
+                "find <search field> - Find task containing search field",
+                "delete <number> - Delete the task");
+    }
+
+    /**
+     * Prints out the greeting message used when the chat started.
+     *
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message greetMessage() {
+        return new Message("Good day there! I'm DUKE\n", "What can I do for you?");
+    }
+
+    /**
+     * Echos the message the user sends for level-1.
+     *
+     * @param s Message user sent.
+     * @return message to be used by either the graphic UI or command line UI.
+     * @Deprecated Level-2
+     */
+    public Message echoMessage(String s) {
+        return new Message(s);
+    }
+
+    /**
+     * Prints out the exit message when chat ends.
+     *
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message exitMessage() {
+        return new Message("Farewell! Hope to see you again.");
+    }
+
+    /**
+     * Prints out chat error message when command is not recognized.
+     *
+     * @return message to be used by either the graphic UI or command line UI.
+     */
+    public Message chatErrorMessage() {
```

Comment:
Not a coding standard issue, but you may want to abstract out the list of commands since both chatErrorMessage( ) and importTaskErrorMessage( ) uses it. If not a developer may end up updating one without updating the other.

Reply:
Thanks for the suggestion, I will have it updated base on your suggestions.